app/models/errors/httpErrors.scala

```diff
@@ -35,8 +35,9 @@ case class UnexpectedStatusError(code: Int, errorResponse: String) extends HttpE
     s"[UnexpectedStatusError]- RESPONSE status: $code, body: $errorResponse"
 }
 
-case object MultipleErrors extends HttpError {
-  override val message: String = "Received multiple errors."
+case class MultipleErrors(code: String, errorResponse: String) extends HttpError {
```

Comment:
Same question here as in your parallel PR on view-vat-returns regarding error _code_ type consistency.